lib/bunyan.js

```diff
@@ -32,8 +32,8 @@ const INTERNAL_DEBUG = false; // Set this to true to enable internal dev/logging
 let xxx, safeJsonStringify, mv, RotatingFileStream;
 
 if (INTERNAL_DEBUG) {
-  xxx = (s) => { // internal dev/debug logging
-    const args = ['XXX: ' + s].concat(Array.prototype.slice.call(arguments, 1));
+  xxx = (s, ...rest) => {
```

Comment:
`...arguments` or `...parameters` is more readable, but it's just a nitpick (since the function is going to be removed anyway).